packages/driver/src/cy/net-stubbing/add-command.ts

```diff
@@ -137,6 +139,16 @@ function validateRouteMatcherOptions (routeMatcher: RouteMatcherOptions): { isVa
     }
   }
 
+  if (routeMatcher.matchUrlAgainstPath) {
+    return err(`\`matchUrlAgainstPath\` was removed in Cypress 7.0.0 and should be removed from your tests. Your tests will run the same. For more information, visit https://on.cypress.io/migration-guide`)
+  }
+
+  for (const prop in routeMatcher) {
+    if (!allRouteMatcherFields.includes(prop)) {
+      return err(`An unknown \`RouteMatcher\` property was passed: \`${String(prop)}\`\n\nValid \`RouteMatcher\` properties are: ${allRouteMatcherFields.join(', ')}`)
+    }
+  }
```

Comment:
does this error add the `docsUrl` property so users can go straight to the docs for `cy.intercept()`?

Reply:
working on a PR here https://github.com/cypress-io/cypress/pull/15711